hops/serving.py

```diff
@@ -160,22 +158,23 @@ def _start_or_stop_serving_rest(serving_id, action):
                                                             response.reason, error_code, error_msg, user_msg))
 
 
-def create_or_update(artifact_path, serving_name, serving_type="TENSORFLOW", model_version=1,
+def create_or_update(serving_name, artifact_path, model_version=1, model_server=None, kfserving=False,
                              batching_enabled = False, topic_name="CREATE",  num_partitions = 1, num_replicas = 1,
-                             instances = 1, kfserving=False):
+                             instances = 1):
     """
     Creates a serving in Hopsworks if it does not exist, otherwise update the existing one.
```

Comment:
Maybe we should add a note that it infers what the model_server is.

Reply:
Done.